### PR TITLE
[measure-tools][breakdown-trees][tree-widget] Updating to core@4.0.0-dev.104

### PR DESCRIPTION
**Pull request by Jake-Screen**
Updating measure-tools, breakdown-trees, and tree-widget core dependencies to 4.0.0-dev.104 and bumping to a prerelease

### CODE REVIEW

**Comment by saskliutas** [comment]
@Jake-Screen pushed fix for tree-widget tests.

**Comment by Jake-Screen** [comment]
> @Jake-Screen pushed fix for tree-widget tests.

Thanks!

File: common/changes/@itwin/breakdown-trees-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/breakdown-trees-react",
+      "comment": "Updating to core@4.0.0-dev.104",
```
Comment (aruniverse) [inline common/changes/@itwin/breakdown-trees-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json:5]:
```suggestion
      "comment": "Updating to iTwin.js core 4.0.0",
```

File: common/changes/@itwin/breakdown-trees-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      "type": "patch"
```
Comment (aruniverse) [inline common/changes/@itwin/breakdown-trees-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json:6]:
```suggestion
      "type": "none"
```

File: common/changes/@itwin/measure-tools-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/measure-tools-react",
+      "comment": "Updating to core@4.0.0-dev.104",
```
Comment (aruniverse) [inline common/changes/@itwin/measure-tools-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json:5]:
```suggestion
      "comment": "Updating to iTwin.js core 4.0.0",
```

File: common/changes/@itwin/measure-tools-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+      "type": "patch"
```
Comment (aruniverse) [inline common/changes/@itwin/measure-tools-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json:6]:
```suggestion
      "type": "none"
```

File: common/changes/@itwin/tree-widget-react/jake-measure-breakdown-core-update_2023-05-12-14-03.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/tree-widget-react",
+      "comment": "Updating to core@4.0.0-dev.104",
```
Comment (aruniverse) [inline common/changes/@itwin/tree-widget-react/jake-measure-breakdown-core-update_2023-05-12-14-03.json:5]:
```suggestion
      "comment": "Updating to iTwin.js core 4.0.0",
```

**Review by saskliutas — APPROVED** [review]

File: packages/itwin/breakdown-trees/package.json [hunk 6]
```diff
@@ -105,19 +105,19 @@
...
-    "@itwin/core-bentley": "^3.7.0",
-    "@itwin/core-common": "^3.7.0",
-    "@itwin/core-frontend": "^3.7.0",
-    "@itwin/core-geometry": "^3.7.0",
+    "@itwin/core-bentley": "^4.0.0-dev.104",
+    "@itwin/core-common": "^4.0.0-dev.104",
+    "@itwin/core-frontend": "^4.0.0-dev.104",
+    "@itwin/core-geometry": "^4.0.0-dev.104",
     "@itwin/core-react": "^4.0.0",
     "@itwin/measure-tools-react": "workspace:^0.11.0",
```
Comment (aruniverse) [inline packages/itwin/breakdown-trees/package.json:117]:
```suggestion
    "@itwin/measure-tools-react": "workspace:^0.12.0-dev.0",
```

File: packages/itwin/breakdown-trees/package.json [hunk 7]
```diff
@@ -105,19 +105,19 @@
...
-    "@itwin/presentation-common": "^3.7.0",
+    "@itwin/presentation-common": "^4.0.0-dev.104",
     "@itwin/presentation-components": "^4.0.0",
-    "@itwin/presentation-frontend": "^3.7.0",
+    "@itwin/presentation-frontend": "^4.0.0-dev.104",
     "@itwin/tree-widget-react": "workspace:^0.9.0",
```
Comment (aruniverse) [inline packages/itwin/breakdown-trees/package.json:121]:
```suggestion
    "@itwin/tree-widget-react": "workspace:^0.10.0-dev.0",
```

**Comment by aruniverse** [comment]
please also update the readme at the root

File: common/changes/@itwin/breakdown-trees-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json (hunk 1 above)
Comment (Jake-Screen) [reply common/changes/@itwin/breakdown-trees-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json]: changed

File: common/changes/@itwin/breakdown-trees-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json (hunk 2 above)
Comment (Jake-Screen) [reply common/changes/@itwin/breakdown-trees-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json]: changed

File: common/changes/@itwin/measure-tools-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json (hunk 4 above)
Comment (Jake-Screen) [reply common/changes/@itwin/measure-tools-react/jake-measure-breakdown-core-update_2023-05-12-13-16.json]: changed

File: common/changes/@itwin/tree-widget-react/jake-measure-breakdown-core-update_2023-05-12-14-03.json (hunk 5 above)
Comment (Jake-Screen) [reply common/changes/@itwin/tree-widget-react/jake-measure-breakdown-core-update_2023-05-12-14-03.json]: changed

File: packages/itwin/breakdown-trees/package.json (hunk 7 above)
Comment (Jake-Screen) [reply packages/itwin/breakdown-trees/package.json]: changed

**Pull request merged**